### PR TITLE
DS3: Update game page

### PR DESCRIPTION
**Pull request by Zunawe**
## What is this fixing or adding?

Rewrites the DS3 game page to be more in line with current settings and better describe certain mechanics.

## If this makes graphical changes, please attach screenshots.

Capture

### CODE REVIEW

**Comment by Zunawe** [comment]
@Marechal-L do you have any comments about the changes? It's mostly just rewording the existing description to be more accurate to the options.

File: worlds/dark_souls_3/docs/en_Dark Souls III.md [hunk 1]
```diff
@@ -7,19 +7,22 @@ config file.
...
-Another option is available to randomize the level of the generated weapons (from +0 to +10/+5).
+Items that can be picked up from static corpses, taken from chests, or earned from defeating enemies or NPCs can be
+randomized. Common pickups like titanite shards or firebombs can be randomized as "progressive" items. That is, the
+location "Titanite Shard #5" is the fifth titanite shard you pick up, no matter where it was from. This is also what
+happens when you randomize Estus Shards and Undead Bone Shards.
 
-To beat the game, you need to collect the 4 "Cinders of a Lord" randomized in the multiworld
-and kill the final boss "Soul of Cinder."
+It's also possible to randomize the upgrade level of weapons and shields as well as their infusions (if they can have
+one), and there are settings that can make the randomized experience more convenient or more interesting, such as
```
Comment (nicholassaylor) [inline worlds/dark_souls_3/docs/en_Dark Souls III.md:16]:
```suggestion
It's also possible to randomize the upgrade level of weapons and shields as well as their infusions (if they can have
one.) Additionally, there are settings that can make the randomized experience more convenient or more interesting, such as
```

Comment (nicholassaylor) [reply worlds/dark_souls_3/docs/en_Dark Souls III.md:16]: Just a small suggestion for sentence flow. Besides this, looks good!

**Review by nicholassaylor — CHANGES_REQUESTED** [review]

File: worlds/dark_souls_3/docs/en_Dark Souls III.md (hunk 1 above)
Comment (Zunawe) [reply worlds/dark_souls_3/docs/en_Dark Souls III.md:16]: Does the period belong inside the parentheses? It looks odd to me, but I know that's the rule for quotation marks.

Comment (nicholassaylor) [reply worlds/dark_souls_3/docs/en_Dark Souls III.md:16]: Mixed up rules for parentheses, suggestion is now correct

Comment (Zunawe) [reply worlds/dark_souls_3/docs/en_Dark Souls III.md:16]: Ah, pointed out in Discord. Sounds good to me.

**Review by nicholassaylor — APPROVED** [review]
LGTM, proofread and checked

**Comment by Br00ty** [comment]
everything looks good to me!

**Review by black-sliver — APPROVED** [review]

**Pull request merged**